examples/python/standalone/opensearch-cloaked-hybrid-search.py

```diff
@@ -0,0 +1,251 @@
+        book["tenant_id"] = tenant_id
+        operations.append(book)
+    bulk_resp = client.bulk("\n".join(map(json.dumps, operations)))
+
+    # Run a hybrid query
+    title_query_embedding = model.encode("python programming").tolist()
+    # `generate_query_vectors` returns a list because the secret involved may be in rotation.
+    encrypted_title_query_embeddings = await sdk.vector().generate_query_vectors(
+        {"title": alloy.PlaintextVector(title_query_embedding, "book_index", "")},
+        metadata,
+    )
+    embedding_queries = [
+        {"knn": {"title_vector": {"vector": title_embedding.encrypted_vector, "k": 5}}}
+        for title_embedding in encrypted_title_query_embeddings["title"]
+    ]
+    search_query = {
+        "size": 5,
+        "query": {
+            "bool": {
+                "filter": {"term": {"tenant_id.keyword": tenant_id}},
+                "should": [
+                    {"match": {"summary": "python programming"}},
+                ]
+                + embedding_queries,
+            }
+        },
+    }
+    response = client.search(index="book_index", body=search_query)
+
+    # Response through Cloaked Search with all results decrypted
+    pretty_response(response)
+
+    # Take a look at the OpenSearch index directly to see what an over-curious admin or someone who exfiltrated
+    # the index would see.
+    document_ids = [r["_id"] for r in response["hits"]["hits"]]
+    bypass_client = OpenSearch(
+        hosts=[{"host": "localhost", "port": 9200}],
+        use_ssl=False,
+        verify_certs=False,
+        ssl_assert_hostname=False,
+        ssl_show_warn=False,
+    )
+    bypass_response = bypass_client.search(
+        index="book_index", body={"size": 5, "query": {"terms": {"_id": document_ids}}}
+    )
+
+    pretty_encrypted_response(bypass_response)
+
+
+asyncio.run(main())
+
+
+# Example response:
+#
+# ID: 7beZW40B2fCVUOIdvs_x
+# Publication date: 2019-05-03
+# Title: Python Crash Course
+# Publisher: no starch press
+# Summary: A fast-paced, no-nonsense guide to programming in Python
+
+# ID: 8beZW40B2fCVUOIdvs_x
+# Publication date: 2018-12-04
+# Title: Eloquent JavaScript
+# Publisher: no starch press
+# Summary: A modern introduction to programming
+
+# ID: 8LeZW40B2fCVUOIdvs_x
+# Publication date: 2015-03-27
+# Title: You Don't Know JS: Up & Going
+# Publisher: oreilly
+# Summary: Introduction to JavaScript and programming as a whole
+
+# ID: 7LeZW40B2fCVUOIdvs_x
+# Publication date: 2019-10-29
+# Title: The Pragmatic Programmer: Your Journey to Mastery
+# Publisher: addison-wesley
+# Summary: A guide to pragmatic programming for software engineers and developers
+
+# ID: 9beZW40B2fCVUOIdvs_y
+# Publication date: 2012-06-27
+# Title: Introduction to the Theory of Computation
+# Publisher: cengage learning
+# Summary: Introduction to the theory of computation and complexity theory
+
+# ID: 7LeZW40B2fCVUOIdvs_x
```

Comment:
imo there should be some sort of output before these 5 indicating that these are now querying against OpenSearch directly, which is why they're duplicated IDs and encrypted fields

Reply:
Ya, this is a good suggestion.